pom.xml

```diff
@@ -21,11 +21,15 @@
   </licenses>
   <properties>
     <powermock.version>1.6.2</powermock.version>
-    <jenkins.version>1.580.1</jenkins.version>
```

Comment:
It gets inherited from the plugin POM, so 1.625.3:
https://github.com/jenkinsci/plugin-pom/blob/plugin-2.7/pom.xml#L50

However, a change in plugin POM version for whatever reason would likely also change this, so it should remain explicit IMO.